packages/mdc-elevation/_mixins.scss

```diff
@@ -95,3 +220,16 @@
 ) {
   @return #{$mdc-elevation-property} #{$duration} #{$easing};
 }
+
+///
+/// Sets the elevation overlay transition value.
+///
+/// @param {String} $duration - The duration of the transition.
+/// @param {String} $easing - The easing function for the transition.
+/// @return {String}
+///
+@function mdc-elevation-overlay-transition-value(
```

Comment:
Can we move this into `_functions.scss`?

Reply:
Sure thing! I'll move all the functions in there.